app/app/src/main/java/tech/ziasvannes/safechat/presentation/chat/ChatViewModel.kt

```
@@ -82,7 +85,39 @@ constructor(
+                                                                            tech.ziasvannes.safechat.data.repository.EncryptionRepositoryImpl)
+                                                                    ?.decryptIncomingMessage(
+                                                                            senderPublicKeyBase64 =
+                                                                                    contact.publicKey,
+                                                                            encryptedContent =
+                                                                                    message.encryptedContent,
+                                                                            iv = message.iv
+                                                                    )
+                                                        }
+                                                        ?.also { atLeastOneDecrypted = true }
+                                                        ?: message.content
+                                            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)